app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,17 @@
+class MoviesController < ApplicationController 
+  
+  def show 
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors 
+  end
+
+  def create
+    @movie = Movie.find(params[:id])
+    actor = Actor.find_by(id: params[:actor_id])
+
+    if actor && !@movie.actors.include?(actor)
```

Comment:
I mention this in the model as well, but this isn't quite the right relationship we're looking for. Since you've created a joins table, you can instead create record in that table (ActorMovies.create).